Content.Shared/Wires/WiresPanelComponent.cs

```diff
@@ -28,11 +28,26 @@ public sealed partial class WiresPanelComponent : Component
     [DataField("screwdriverCloseSound")]
     public SoundSpecifier ScrewdriverCloseSound = new SoundPathSpecifier("/Audio/Machines/screwdriverclose.ogg");
 
+    /// <summary>
+    ///     A verbal description of the wire panel's current security level
+    /// </summary>
     [AutoNetworkedField]
     public string? WiresPanelSecurityExamination = default!;
 
+    /// <summary>
+    ///     Determines whether the wiring is accessible to hackers or not
+    /// </summary>
     [AutoNetworkedField]
     public bool WiresAccessible = true;
+
+    /// <summary>
+    ///     Determines whether the device can be welded shut or not
+    /// </summary>
+    /// <remarks>
+    ///     Should be set false when you need to weld/unweld something to/from the wire panel
+    /// </remarks>
+    [AutoNetworkedField]
+    public bool WeldingAllowed = true;
 }
```

Comment:
Why do these fields just double the prototype?

Reply:
The security prototypes determine what the appropriate values should be for these variables for each security level. The variables on the wire panel hold a copy of the values for the current security level on the door and get updated when the security level changes. Do you want a prototype variable for the current security level on the wires panel component instead, and read its values when needed?